pkg/hubble/relay/observer/server.go

```diff
@@ -0,0 +1,210 @@
+// Copyright 2020 Authors of Cilium
```

Comment:
this file is still named `server.go` but in the observer package

Reply:
This file contains the `Server` struct which implements the `observerpb.ObserverServer` interface and the `NewServer()` method to instantiate it. I thought it made sense to keep the filename as `server.go`.

Reply:
Discussed offline, `server.go` actually seems like a decent choice here, since we still have the `observer.go` file containing some package functions. Let's keep `server.go` then.